### ISSUE TITLE
Improve button labels for clarity.

### ISSUE THREAD

**Issue by LivingSynthesis** [Unreal-Dan/lightshow.lol#47]
![image](https://github.com/user-attachments/assets/20971adc-247e-4adc-b4f1-dde75f8518bd)

New button name suggestions:

Connect -> Good as is
Pull -> Load/Load Save
Push -> Overwrite
Transmit -> Good as is

These might be good to put in a hamburger menu as well
Add -> Good as is
Export -> copy
Import -> paste
Link -> Good as is
Share -> Save (idk this one is complicated)

This naming issue could also be circumvented with mouse-over tooltips

**Comment by Unreal-Dan** [comment]
Yeah I like the copy/paste instead of export/import, they could also just copy it to the clipboard and throw a little green notification saying 'Mode Copied to Clipboard!' or when you click paste it will just say something like 'Mode Pasted from Clipboard!' and add the new mode.  (I don't think they need modals to pop up...)

We can add cltr+c and ctrl+v keypress handlers, but it gets a bit nasty when you want to make them work on other things like the other 'windows' say copy/pasting an led or a color.  The reason it gets nasty is because there is no concept of 'focus' right now, because they aren't really 'windows' they're just boxes on the screen that look like windows.

I'm pretty sure some javascript could make them behave more like windows by detecting clicks in areas like the 'toolbar' part of the window then making it change absolute position with your cursor movement (drag the window). 

----

I'm not a fan of Push -> Overwrite, I would prefer maybe 'Pull Modes' and 'Push Modes'.  I understand the interest in the word overwrite though, which sparked an idea, how about 'Push Modes' has a modal that says

> Are you sure you want to Push Modes to your device?  This will **overwrite** the modes currently on your device!
> [ ] Do not show this message again
> [Yes, overwrite my modes] [No, don't touch my modes]

That way they are always warned about the potentially dangerous operation, because yeah, this should be a protected operation. It ended up as a happy accident that I had to move it into a hamburger for lack of space -- because this made sure you can can't misclick it as easily. But it still needs a confirmation for sure.

----

Share -> Save (idk this one is complicated)

Nah I like share and I think it's quite intuitive, you're sharing it with the community. 

Worst case a confused user clicks it and end up on the community, it will be instantly obvious what the button does.

Right now it lands them on a login page because the upload is login protected, but I think what I'll do is have it redirect to the showcase or home page if they aren't logged in when they click share.
